cli/ceph/orch/upgrade.py

```diff
@@ -0,0 +1,51 @@
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class Upgrade(Cli):
+    def __init__(self, nodes, base_cmd):
```

Comment:
Won't these methods take longer time for execution.

Reply:
No Vaibhav

Only method "upgrade start" takes times for same we have waiter method in test script.

Test Script:
https://github.com/red-hat-storage/cephci/pull/2552/files#diff-26e428b3fc56a702afccfc8fb33a7f6ad39c9d99ba291eba408bb2cdcb738ebaR21